documentation/docs/faq.md

```diff
@@ -10,4 +10,27 @@ keywords:
 - IOTA Identity
 ---
 
-TODO: Add list of frequently asked questions and there answers
\ No newline at end of file
+This page contains frequently asked questions regarding the Identity Library and Self Sovereign Identity in general.
+
+### What programming languages are supported by the identity library?
+Please see the "Programming Languages" section, to find out which languages are available.
+### Do I need to have IOTA tokens to start building with identity?
+At the moment you don't need IOTA tokens to create and manage identities, although we are looking at opportunities to utilize the token in the future. 
+### How do I prove control over my DID?
+Control over an identity is ultimately tied to the control over cryptographic key material (something you have).
+### How can I store my private keys?
+Theoretically you can store the keys however you like. We provide a secure default where possible using a best effort software key storage. For better guarantees you could look into hardware based key storage.
+### Do I need a Permanode to use IOTA Identity?
+Current you need to have access to a Permanode (a node with all the history of the tangle) to correctly resolve identities. The trustworthiness of this node is very important, since a malicious node could flat out respond with made up data, so ideally you run that node yourself or make sure the party supplying the node is trustworthy.
```

Comment:
Slightly correct the malicious node part: they can't make up data since it's cryptographically verifiable.

```suggestion
### Do I need a Permanode to use IOTA Identity?
You can get started without one but currently you do require access to a Permanode (a node which stores the entire history of the Tangle) to reliably resolve identities in production. The trustworthiness of this node is very important, since a malicious node could respond with outdated identities, so ideally you should run one yourself or ensure the party supplying the node is trustworthy.
```